model.yml

```
@@ -1,5 +1,5 @@
-short_name: template-model
-version: 1.0.0
-binary: project
+short_name: prostate
+version: 2.0.0
```

```suggestion
version: 1.0.0
```
Or is there a version 1.0.0 already ??

---

I consider the version that we included in the PhysiBoSS 2.0 paper as v1, but do as you wish